pkg/urbit/daemon/main.c

```diff
@@ -991,6 +993,73 @@ _cw_intr_win(c3_c* han_c)
 }
 #endif
 
+/* _cw_eval_commence(): initialize and run the hoon evaluator
+*/
+static void
+_cw_eval_commence(c3_i argc, c3_c* argv[])
+{
+  //Read from stdin until an EOF is recieved
+  c3_c* line = NULL;
+  size_t len = 0;
+  ssize_t nread;
+  c3_c* evl_c = "";
+
+  while((nread = getline(&line, &len, stdin)) != -1)
+  {
+     len = asprintf(&evl_c, "%s%s", evl_c, line);
+  }
+
+  free(line);
+
+  //printf("%s", evl_c);
+
+  //Initialize the Loom and load the Ivory Pill
+  c3_d          len_d = u3_Ivory_pill_len;
+  c3_y*         byt_y = u3_Ivory_pill;
+  u3_cue_xeno*  sil_u;
+  u3_weak       pil;
+
+  u3C.wag_w |= u3o_hashless;
+  u3m_boot_lite();
+  sil_u = u3s_cue_xeno_init_with(ur_fib27, ur_fib28);
+  if ( u3_none == (pil = u3s_cue_xeno_with(sil_u, len_d, byt_y)) ) {
+    printf("*** fail _setup 1\n");
+    exit(1);
+  }
+  u3s_cue_xeno_done(sil_u);
+  if ( c3n == u3v_boot_lite(pil) ) {
+    printf("*** fail _setup 2\n");
+    exit(1);
+  }
```

Comment:
This could was copied from a unit test, it'd be better to use the error printfs from the equivalent in `king.c` (`_king_boot_ivory()`). Even better, declare that function so it's accessible here and then just call it.

Reply:
Changed to be the same error printfs